users/api/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views import ProfileDetailAPIView, ProfileUpdateAPIView
+
+urlpatterns = [
+    path('profile/<user>/', ProfileDetailAPIView.as_view(), name='detail'),
+    path('profile/<user>/update/', ProfileUpdateAPIView.as_view(), name='update'),
```

Comment:
why is `<user>` required in the `update` call. It should be just `profile/update` that automatically takes the user which is authenticated.

Reply:
Yes I will fix this in a future commit!